pommerman/envs/v0.py

```diff
@@ -238,10 +253,15 @@ def render(self, mode='human', close=False, record_pngs_dir=None, record_json_di
                 if agent.has_user_input():
                     self._viewer.window.push_handlers(agent)
         else:
-            self._viewer.imshow(img)
+            self._viewer.set_board(self._board)
+            self._viewer.set_agents(self._agents)
+            self._viewer.set_step(self._step_count)
+            self._viewer.render()
 
         if record_pngs_dir:
-            Image.fromarray(img).save(os.path.join(record_pngs_dir, '%d.png' % self._step_count))
+            path = os.path.join(record_pngs_dir, '%d.png' % self._step_count)
+            # self._viewer.save(path)
+            Image.fromarray(img).save(path)
```

Comment:
Seems like you can get rid of this and use self._viewer's save if you include the img, no?